resources/guis.yml

```diff
@@ -168,15 +168,15 @@ guis:
   - Mac
   price: $49.99, free evaluation
   license: Proprietary
-  order: 18
+  order: 2
```

Comment:
That's an awfully big jump. I wonder if the name "fork" is so generic that the trends search is picking up unrelated hits.